daemon/src/projection.rs

```diff
@@ -119,6 +119,8 @@ pub struct Cfd {
     #[serde(with = "::bdk::bitcoin::util::amount::serde::as_btc")]
     pub accumulated_fees: SignedAmount,
 
+    // TODO: This is currently always the taker leverage! This should be clear
+    /// The taker leverage
     pub leverage: Leverage,
```

Comment:
🙃 You could rename the struct field to `taker_leverage` and use `#[serde(rename = "leverage")]` if you don't want to break the API.